docs/developersetup.md

```diff
@@ -0,0 +1,43 @@
+# Developer Setup
```

Comment:
Please update the README.md file:

https://github.com/mellychat/webapp/blob/mainline/README.md

I think you can remove the TODO related to this PR and also link over to this document in a "Next Steps" section.

Reply:
I see it's done in https://github.com/mellychat/webapp/pull/88. 

In the future you can do small changes like that within this PR. If you need to handle a suggestion in a different PR, try to link it to the conversation thread so you're reviewer knows to check the other PR.